apps/cowswap-frontend/src/legacy/state/swap/extension.ts

```diff
@@ -55,6 +54,8 @@ export function useTradeExactInWithFee({
 
   // external price output as Currency
   const outputAmount = stringToCurrency(quote.price.amount, outputCurrency)
+  const partnerFeeAmount = partnerFee ? outputAmount.multiply(bpsToPercent(partnerFee.bps)) : undefined
```

Comment:
Can we use `0` for `partnerFeeAmount`, so there's no need for conditional and it's safe to always subtract it from `outputAmount`?

Reply:
I would prefer to keep it more explicit.
If we set 0 here, then it will be difficult to distinguish if the partner fee is not set or set to 0